back-end/pom.xml

```diff
@@ -77,6 +77,13 @@
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.projectlombok</groupId>
```

Comment:
Are we using lombok? if we absolutely need lombok, can we use a newer version because I keep getting this error `Illegal reflective access by lombok.javac.apt.LombokProcessor`

Reply:
I am going to remove it and just use getters because I am getting some issues too

Reply:
When you remove it, I'll test it locally and approve it

Reply:
Removed